qa/L0_long_running_stress/stress.py

```diff
@@ -514,6 +546,8 @@ def stress_thread(name, seed, test_duration, correlation_id_base):
                 # exercise the idle sequence path of the sequence
                 # scheduler
                 if choice < 0.33:
+                    count_test_case("sequence_no_end")
```

Comment:
For the sequences also count the number of individual requests. In the table you will then have a column for the number of times that test case was chosen ("Count"), and another for how many total requests were sent for that case ("Request Count")

Reply:
Got it, will add new column for that.

Reply:
I think it would be better to use "Test Count" instead of "Count" since Count is not as descriptive.
>  the number of times that test case was chosen ("Count")